graylog2-web-interface/src/components/lookup-tables/DataAdapterTableEntry.test.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import * as React from 'react';
+import { render, screen } from 'wrappedTestingLibrary';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import { asMock } from 'helpers/mocking';
+
+import { DATA_ADAPTER, mockedUseScopePermissions } from './fixtures';
+import DataAdapterTableEntry from './DataAdapterTableEntry';
+
+jest.mock('hooks/useScopePermissions', () => (mockedUseScopePermissions));
+
+const renderedDataAdapter = (scope: string) => {
+  const auxDataAdapter = { ...DATA_ADAPTER };
+  auxDataAdapter._scope = scope;
+
+  return render(
+    <Router>
+      <DataAdapterTableEntry adapter={auxDataAdapter} error={null} />
+    </Router>,
+    {
+      container: document.body.appendChild(document.createElement('table')),
+    },
+  );
+};
```

Comment:
There should be no need to render `Router`, since it is already part of the `DefaultProviders` we use for `render` from `wrappedTestingLibrary`.

Reply:
I think the following approach is a little bit easier to read:
```
return render(
  <table>
    <DataAdapterTableEntry adapter={auxDataAdapter} error={null} />
  </table>,
);
```

(in this case and in the following cases which are similar)